PhpAmqpLib/Channel/AMQPChannel.php

```diff
@@ -342,6 +341,10 @@ public function exchange_declare(
         $arguments = null,
         $ticket = null
     ) {
+        if (!$this->is_open) {
+            throw new AMQPRuntimeException('Channel connection is closed.');
```

Comment:
What does "channel connection" mean? What specifically is closed, a channel or a connection?